CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 Entries are listed in reverse chronological order.
 
-## Unreleased
+## 0.4.1
 
-* Migrate to `group` 0.13, `jubjub` 0.10, `pasta_curves` 0.5
+* Add Pallas and Jubjub ciphersuites and FROST support (#33)
+* Migrate to `group` 0.13, `jubjub` 0.10, `pasta_curves` 0.5 (#44)
```

Comment:
This needs to be an `0.5.0` release, because migrating to `group 0.13` changes the public API.

Reply:
Good point, thanks!

Reply:
It's a bit tricky because `group` doesn't appear in any _documented_ public API, but it appears in the associated types of `pub trait Sealed` (which is not implementable outside the crate due to being sealed, but is technically part of the public API).